### ISSUE TITLE
Agile User Stories Persona Type Selection

### ISSUE THREAD

**Issue by byklyci** [bounswe/bounswe2018group9#56]
Guys, as you know after our meeting [#Meeting 4](https://github.com/bounswe/bounswe2018group9/wiki/Meeting-%234-(26.02.2018)), we select 3 type of person but one of them was not clear please add your comment here to solve this problem.

**Comment by byklyci** [comment]
One of them is ordinary,busy,asocial(Muhittin)
One of them is rich,wants to be known,high society(İremsu)
But the third one is fuzy.Lets talk about the our persona types.

**Comment by nazmicancalik** [comment]
I think we should start doing already prepared personas, and while we are creating them, I'm sure we can find good ideas.

**Comment by serkan-ozel-2015400123** [comment]
@bounswe/bounswe2018group9 Let's the last one be a event organizator. He or she can be an ordinary person who wants to be popular through internet. He or she can use our system to advertise his or her event and also give information about it.

**Comment by okyksl** [comment]
@kemaltulum could you share your ideas on our last user? I remember that we couldn't explore your perspective on the last user on the meeting.

**Comment by gumityolcu** [comment]
I think it should be an amateur trying to get recognition because of two reasons:

1- This is one way of using our platform and we should prepare a scenario considering an amateur artist/performer so that our product meets probable acceptance criteria.

2- We don't have any user persona that creates events, if we don't user scenario process will not help us about realising the event-creation-related parts of our system.

**Comment by kemaltulum** [comment]
I think, there has always been an ambiguity with the organizer perspective in our project, which is not related with us but the assignment given. @gumityolcu I agree with you, an artist, a popular one, can utilize our product to invite people his/her event, which is somewhat fun club like use case.

On the other hand, probably there will be no explicit entity like organizer in the project, since the way the project given puts it in this way enabling each user creates events. 

@okyskl My perspective was the people having crazy interests. Actopus can make them together facilitating their organization with nice features.

In short, we have such a user with crazy interests and considering İremsu as an organizer, since she also has same desires like being popular and making people engaged what she is doing

**Comment by okyksl** [comment]
I see that we formed a consensus on `Şeyda`'s and `Ceyda`'s - former `İremsu` - characters. `Şeyda` will take on the role of a event creator and `Ceyda` will take on the role of a frequent event participant. I'm closing the issue as the distinction between personas have been made.
